Form-Controls/index.html

```diff
@@ -13,15 +13,67 @@ <h1>Product Pick</h1>
     </header>
     <main>
       <form>
+          <p>
+            <label for="name">Name:</label>
+            <input type="text" id="name" name="user_name" required minlength="2" maxlength="50" />
+
+          </p>
+          <p>
+            <label for="mail">Email:</label>
+            <input type="email" id="mail" name="user_email" required/>
+          </p>
+          <p>
+            <label for="colour"> What colour should this T-shirt be? </label> <br>
+              <input type="radio" id="green" name="Tshirt_colour" value="Green">
+                <label for="green"> Green</label>
+                <input type="radio" id="red" name="Tshirt_colour" value="Red">
+                <label for="red"> Red</label>
+                <input type="radio" id="yellow" name="Tshirt_colour" value="Yellow">
+                <label for="yellow"> Yellow</label>               
+          </p>
+          <p>
+            <label for="size"> Please choose a size. </label><br>
```

Comment:
Your labels in inputs below are not connected correctly. Please, think what should correctly connect label with input.

Reply:
Never mind, your input was before the label and hence my concussion. All good. Just on line 35 I don't think you need `for` attribute or `label` tag. But it is minor issue.